cli/src/commands/prev.rs

```diff
@@ -107,14 +95,16 @@ pub(crate) fn cmd_prev(
         [target] => target,
         [] => {
             return Err(user_error(format!(
-                "No ancestor found {offset} commit{} back",
-                if offset > 1 { "s" } else { "" }
+                "No ancestor found {} commit{} back",
+                args.offset,
```

Comment:
nit: either revert this change or update `cmd_next()` to use `args.offset`

Reply:
Thanks. I decided to delete the offset variable in next.rs as well. There isn't a good reason for it to exist and I think it's clearer if we just access the offset through the args struct.